doc/changelog.rst

```diff
@@ -162,6 +162,12 @@ Breaking Changes in 4.0
   projection will now return the entire document, not just the ``"_id"`` field.
 - ``MongoClient()`` now raises a :exc:`~pymongo.errors.ConfigurationError`
   when more than one URI is passed into the ``hosts`` argument.
+- ``MongoClient()`` now raises an exc:`~pymongo.errors.InvalidURI` exception
+  when it encounters unescaped percent signs in username and password.
```

Comment:
in username or password when parsing MongoDB URIs.

Reply:
Done.